beta/src/pages/learn/your-first-component.md

```diff
@@ -114,24 +114,19 @@ return (
 
 <Gotcha>
 
-Without parentheses, any code on the lines after `return` [will be ignored](https://stackoverflow.com/questions/2846283/what-are-the-rules-for-javascripts-automatic-semicolon-insertion-asi)!
+ब्रैकेट के बिना, `return` के बाद लाइनों पर लिखा कोई भी कोड [इग्नोर हो जाएगा](https://stackoverflow.com/questions/2846283/what-are-the-rules-for-javascripts-automatic-semicolon-insertion-asi)!
 
 </Gotcha>
 
-## Using a component {/*using-a-component*/}
+## एक कौम्पोनॅन्ट का उपयोग करना {/* using-a-component */}
```

Comment:
`{/* using-a-component */}` => `{/*using-a-component*/}`

Let's not make any changes to these. These will be used for reflinks